.github/workflows/screener-run.yml

```diff
@@ -132,20 +175,38 @@ jobs:
 
   screener-react-components:
     runs-on: 'ubuntu-latest'
-
     name: Screener @fluentui/react-components
-
     steps:
       - uses: actions/checkout@v3
         with:
-          ref: ${{github.event.workflow_run.head_branch}}
           fetch-depth: 0
 
       - uses: actions/setup-node@v3
         with:
           node-version: 14.18.1
           cache: 'yarn'
 
+      - name: Check if test app artifact deployed
+        uses: actions/github-script@v6
+        id: skip-screener
+        with:
+          script: |
+            let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: 2917202539,
+            });
+            let matchArtifact = allArtifacts.data.artifacts.filter((artifact) => {
+              return artifact.name == "vnext-artifact"
+            })[0];
+            if(matchArtifact !== undefined)
+              return 'false'
+            return 'true'
+          result-encoding: string
+
+      - name: Set env variable for skipping screener
+        run: echo "SKIP_SCREENER_BUILD=${{steps.skip-screener.outputs.result}}" >> $GITHUB_ENV
```

Comment:
Can you please clarify why we need to build artifacts there again?

If I understood it correctly it does following:

```mermaid
graph TD
    CHOICE{Is there artifact?}
    CHOICE --> BUILD[Build artifacts]
    CHOICE --> DOWNLOAD[Download artifact]
    BUILD --> UPLOAD[Upload artifacts]
    START[Start a test]
    UPLOAD --> START
    DOWNLOAD --> START
```

Reply:
the `SKIP_SCREENER_BUILD` variable is false if there has been a storybook artifact uploaded(so we shouldn't skip the screener build part) and true if not and the screener build was skipped; after defining this env variable we can use it to determine whether we should try to download the artifact or not(because it doesn't even exist)
So if there is an artifact(that was built in the previous workflow), we will download the artifact, upload it and then start the screener
If there is not any artifact from the previous workflow, we skip the part where we download and upload it, and we just start the screener run that will proceed to skip that check from the proxy side

Reply:
Oh, nevermind I need one more coffee in morning ☕ 🐱 
Would you mind to rename `SKIP_SCREENER_BUILD` to something like `IS_ARTIFACT_PRESENT` or something similar to indicate that it is related to downloading artifacts rather than building them?

Reply:
sure!